### PR TITLE
KEA-143 DB - Setup DB

### PR DESCRIPTION
**Pull request by doolri1276**
DB - Setup DB

### CODE REVIEW

File: backend/src/api/announcement/content-types/announcement/schema.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
+{
+  "kind": "collectionType",
+  "collectionName": "announcements",
+  "info": {
+    "singularName": "announcement",
+    "pluralName": "announcements",
+    "displayName": "Announcement",
+    "description": ""
+  },
```
Comment (doolri1276) [inline backend/src/api/announcement/content-types/announcement/schema.json:9]: missing blahblah

File: backend/src/api/announcement/content-types/announcement/schema.json [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    },
+    "Private": {
+      "pluginOptions": {
+        "i18n": {
+          "localized": false
+        }
+      },
+      "type": "boolean",
+      "default": false,
+      "required": true
```
Comment (doolri1276) [inline backend/src/api/announcement/content-types/announcement/schema.json:64]: foeofjowjf qfe

File: backend/src/api/announcement/controllers/announcement.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+'use strict';
+
+/**
+ * announcement controller
```
Comment (doolri1276) [inline backend/src/api/announcement/controllers/announcement.js:4]: @hyunghoseo Is this the right approach?

File: backend/src/api/announcement/content-types/announcement/schema.json (hunk 2 above)
Comment (doolri1276) [reply backend/src/api/announcement/content-types/announcement/schema.json:64]: I worked on it

**Review by hyunghoseo — APPROVED** [review]

**Pull request merged**